core/src/test/java/nu/yona/server/device/service/DeviceServiceTest.java

```diff
@@ -221,7 +243,8 @@ public void addDeviceToUser_tryAddDuplicateName_exception()
 	public void deleteDevice_deleteOneOfTwo_userHasOneDevice()
```

Comment:
🙃 `_devicesUpdatedAndDeviceChangeMessageSent`

Reply:
Named it ``deleteDevice_deleteOneOfTwo_userHasOneDeviceBuddyInformed``, in accordance with the other test name: ``updateDevice_rename_renamedBuddyInformed``,